Android/G3MAndroidSDK/src/org/glob3/mobile/specific/SQLiteStorage_Android.java

```diff
@@ -16,19 +21,23 @@
 import org.glob3.mobile.generated.TimeInterval;
 import org.glob3.mobile.generated.URL;
 
-import android.content.ContentValues;
-import android.database.Cursor;
-import android.database.sqlite.SQLiteDatabase;
-import android.database.sqlite.SQLiteOpenHelper;
-import android.graphics.Bitmap;
-import android.graphics.BitmapFactory;
-import android.util.Log;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.Charset;
 
 
 public final class SQLiteStorage_Android
          extends
             IStorage {
 
+   private static final long DB_RAW_DATA_EXTERNAL_THRESHOLD = 1024 * 1024; // 1 MiB
+   private static final String DB_EXTERNAL_TAG = "EXT:";
```

Comment:
Let's make DB_RAW_DATA_EXTERNAL_THRESHOLD an instance field and add a constructor where the threshold could be defined when creating the g3m widget.